pyro/distributions/torch/__init__.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
This file is now much closer to being a simple loop over `dir(torch.distributions)`. It still remains to rename parameters and fix `Multinomial` parameter types.